### PR TITLE
[WIP] Enable cluster-autoscaler

### PR DESCRIPTION
**Pull request by frobware**
This is very much a work-in-progress effort. There are commits here that should disappear.

### CODE REVIEW

**Comment by frobware** [comment]
This change also needs to be for 3.11.

File: roles/openshift_aws/tasks/scale_group.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@
           | combine(openshift_aws_node_group.tags)
           | combine({'deployment_serial': l_deployment_serial, 'ami': openshift_aws_ami_map[openshift_aws_node_group.group] | default(openshift_aws_ami)})
           | combine({'openshift-node-group-config': openshift_aws_node_group.node_group_config | default('unset') }) }}"
+    - Name: "{{ openshift_aws_clusterid }}"
```
Comment (ingvagabund) [inline roles/openshift_aws/tasks/scale_group.yml]: Life saver!!!

**Comment by ingvagabund** [comment]
Had to add more rules and `s/system:cluster-autoscaler/cluster-autoscaler` in the `clusterrole.yml` file: https://github.com/ingvagabund/openshift-ansible/commit/b00195f331de1ff9f8522866f4ca090de2ca2558

After applying the changes on top of this PR, the autoscaler scales up and down \O/.

**Comment by frobware** [comment]
Closing this in favour of https://github.com/openshift/openshift-ansible/pull/9147

**Pull request closed without merging**